back/src/Services/SocketManager.ts

```diff
@@ -614,6 +618,49 @@ export class SocketManager {
         user.socket.write(serverToClientMessage);
     }
 
+    public async handleJoinBBBMeetingMessage(user: User, joinBBBMeetingMessage: JoinBBBMeetingMessage) {
+        const meetingId = joinBBBMeetingMessage.getMeetingid();
+        const meetingName = joinBBBMeetingMessage.getMeetingname();
+
+        const api = BigbluebuttonJs.api(BBB_URL, BBB_SECRET);
```

Comment:
If BBB_URL is empty, we should probably consider:

-  sending an error message to the client (using the ErrorMessage protobuf message)
- logging an error in the pusher logs using console.error

Reply:
I believe it has been addressed by this new commit https://github.com/thecodingmachine/workadventure/pull/1948/commits/fd93899381ef7694caa9b59d75ac43cecf8aaee5